### ISSUE TITLE
GitHub pages support

### ISSUE THREAD

**Issue by XBagon** [NiklasEi/bevy_game_template#23]
Are you interested in supporting GitHub pages for WASM builds?
I wanted to make my game's releases very easy to play.
For the moment I just added 
```yaml
      - name: Deploy to GitHub Pages
        uses: JamesIves/github-pages-deploy-action@v4.2.5
        with:
          branch: gh-pages
          folder: wasm
          clean-exclude: index.html
```
as an additional step of  the release-wasm job.

In the future extending this feature to support multiple (all available) release and not just the most recent on could be very interesting.

**Comment by NiklasEi** [comment]
Nice idea!
I ran into a few issues with the web build setup when I tried this in the release workflow. When running on GH pages, the assets are served under a subpath (your repo name). But, rewriting the asset paths to be relative sadly breaks the normal development workflow with `trunk serve`.

I think a good way of supporting this would be to add a workflow that can be manually triggered and only does the deploy. The build in that workflow can then use relative links for the assets.
